### ISSUE TITLE
 nunit-console /runlist:testlist.txt nunit.tests.dll not working on my side

### ISSUE THREAD

**Issue by somnath4993** [nunit/nunit-console#481]
i am trying to pass testlist.txt to nunit but its without executing showing result all test case are passed and time elapsed., i have written my test cases in specFlow feature file with gherkin language.

**Comment by mikkelbu** [comment]
Hi @somnath4993 
First and foremost, I don't think that nunit3-console supports `runlist`, so I guess that you are using NUnit2. If that is the case then https://github.com/nunit-legacy/nunitv2 is the right repo. 

But for us to be able to help you, you'll need to provide some more information, as it is not clear from the text what the problem is. So:
* How exactly are you calling nunit-console
* What happens (perhaps also supply output)
* What did you expect should happen

**Comment by ChrisMaddock** [comment]
See the below page of breaking changes, `--runlist` has been replaced by `--testlist`.

https://github.com/nunit/docs/wiki/Breaking-Changes#console-features

**Comment by mikkelbu** [comment]
Chris have already answered, but I'm curious 😄 . If you run the console with `--runlist:PATH` as parameter don't you get an error from the console?

```
nunit-console\bin\Release $ .\nunit3-console.exe --runlist:somefile.txt .\nunit.engine.tests.dll
NUnit Console Runner 3.9.0
Copyright (c) 2018 Charlie Poole, Rob Prouse

Invalid argument: --runlist:somefile.txt
```

**Comment by somnath4993** [comment]
sorry i was use testlist only

**Comment by somnath4993** [comment]
![image](https://user-images.githubusercontent.com/14311712/46610301-f9358f00-cb27-11e8-9793-b0404c77553b.png)
here i am sending my command line screenshot, it is not executing test case, just showing test case passed, you can see the time in screenshot

**Comment by mikkelbu** [comment]
No need to be sorry. I was just curious whether we had a problem somewhere. Are you sure that the lines in the `Run.txt` matches the name of the tests? Remember thar the names of the tests should be fully qualified, e.g. `MyNamespace.MyClass.TestA`.

**Comment by ChrisMaddock** [comment]
I agree with Mikkel. You can use the `--explore` flag, to see what your test names should look like. 🙂

**Comment by somnath4993** [comment]
I have use --explore command and its shown all my scenario which I written in my feature file. and similar name I have mention in run.txt file but no luck

**Comment by somnath4993** [comment]
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC01_Create_Login_for_all_required_Users
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC02_Pre_DataSetup_for_batch_execution
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC03_Pre_DataSetup_for_batch_executionTraderLogin1
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC04_Pre_DataSetup_for_batch_executionTraderLogin2
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC05_Pre_DataSetup_for_batch_executionTraderLogin4
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC06_Pre_DataSetup_for_batch_executionTraderLogin5
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC07_VerifyingthefunctionalityofActivatingandModifyingaBid/OfferOrderfromMyBlotterforCDSInstrument

**Comment by somnath4993** [comment]
my run.txt like below

TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC01_Create_Login_for_all_required_Users
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC02_Pre_DataSetup_for_batch_execution
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC03_Pre_DataSetup_for_batch_executionTraderLogin1
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC04_Pre_DataSetup_for_batch_executionTraderLogin2
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC05_Pre_DataSetup_for_batch_executionTraderLogin4
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC06_Pre_DataSetup_for_batch_executionTraderLogin5
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC07_VerifyingthefunctionalityofActivatingandModifyingaBid/OfferOrderfromMyBlotterforCDSInstrument

**Comment by CharliePoole** [comment]
Can you post the --explore output?

**Comment by somnath4993** [comment]
D:\CDS_API_Automation\branches\TPCD_Activate_Modify_Order>"D:\CDS_API_Automation\branches\packages\NUnit.ConsoleRunner.3.9.0\tools\nunit3-console.exe" "D:\CDS_API_Autom
odify_Order.dll" --explore
NUnit Console Runner 3.9.0
Copyright (c) 2018 Charlie Poole, Rob Prouse

Runtime Environment
   OS Version: Microsoft Windows NT 6.1.7601 Service Pack 1
  CLR Version: 4.0.30319.42000

Test Files
    D:\CDS_API_Automation\OutputLibs\TPCD_Activate_Modify_Order.dll

TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC01_Create_Login_For_All_Required_Users
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC02_Pre_DataSetup_For_Batch_Execution
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC03_Pre_DataSetup_For_Batch_ExecutionTraderLogin1
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC04_Pre_DataSetup_For_Batch_ExecutionTraderLogin2
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC05_Pre_DataSetup_For_Batch_ExecutionTraderLogin4
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC06_Pre_DataSetup_For_Batch_ExecutionTraderLogin5
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC07_VerifyingTheFunctionalityOfActivatingAndModifyingABidOfferOrderFromMyBlotterForCDSInstrument
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC08_VerifyTheFunctionalityOfNonActivatingAndReactivatingTheHeldBidOfferOrderFromMyBlotterForCDSInstrument
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC09_ActivatingSamecompanyBidOfferOrderBySamecomptrader
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC10_VerifyTheBidOfferOrderEntryWithPriceNotWithinTheThresholdValueFromMyBlotterForCDS
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC11_VerifyTheBidOfferOrderEntryWithPriceOtherThanPriceTickSizeValueAsConfiguredInAdminConsoleFromMyBlotte
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC12_VerifyingTheFunctionalityOfModifyingABidOrderWithPriceNotWithinTheThresholdLimitsFromMyBlotterForCDSI
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC13_VerifyingTheFunctionalityOfModifyingAOfferOrderWithPriceNotWithinTheThresholdLimitsFromMyBlotterForCD
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC14_VerifyingTheFunctionalityOfModifyingABidOrderOwnedByBrokerWithSizeRangeGreaterThanThresholdValueAndNo
eFromMyBlotter

**Comment by CharliePoole** [comment]
Change the casing in your testlist file so it __exactly__ matches the names in the explore output. For example change this...

```
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC01_Create_Login_for_all_required_Users
```
to this...

```
TPCD_Activate_Modify_Order.FeatureFile.Activate_ModifyFeature.TC01_Create_Login_For_All_Required_Users
```

**Comment by vkulkarniS23** [comment]
I have a similar issue. My casing of the tests matches exactly with the output file. It still doesn't run. Any help will be appreciated.
